pkg/controller/operators/olm/operator.go

```diff
@@ -421,10 +421,33 @@ func (a *Operator) syncOperatorGroups(obj interface{}) error {
 		return fmt.Errorf("casting OperatorGroup failed")
 	}
 
-	if err := a.updateDeploymentAnnotation(op); err != nil {
+	err, targetedNamespaces := a.updateDeploymentAnnotation(op)
+	if err != nil {
 		return err
 	}
 
+	for _, ns := range targetedNamespaces {
+		csvsInNamespace := a.csvsInNamespace(ns.Name)
+		for _, csv := range csvsInNamespace {
+			if csv.Status.Phase == v1alpha1.CSVPhaseSucceeded {
+				newCSV := csv.DeepCopy()
+				newCSV.Status = v1alpha1.ClusterServiceVersionStatus{
+					Message:        "CSV copied to target namespace",
+					Reason:         v1alpha1.CSVReasonCopied,
+					LastUpdateTime: timeNow(),
+				}
+				newCSV.Annotations["OriginalCSV"] = fmt.Sprintf("Namespace:%v, ResourceVersion:%v", csv.GetNamespace(), csv.GetResourceVersion())
```

Comment:
:+1: can we make this more machine readable `<namespace>/<csvname>`?

also we should follow the annotation key convention we used before, I think it would be `olm.originalCSV`?